src/mame/trs/coco12.cpp

```diff
@@ -278,6 +288,109 @@ static INPUT_PORTS_START( coco )
 INPUT_PORTS_END
 
 
+//-------------------------------------------------
+//  INPUT_PORTS( deluxecoco_keyboard )
+//-------------------------------------------------
+//
+//  Deluxe CoCo keyboard
+//
+//         PB0 PB1 PB2 PB3 PB4 PB5 PB6 PB7
+//    PA6: Ent Clr Brk Alt Ctr F1  F2 Shift
+//    PA5: 8   9   :   ;   ,   -   .   /
+//    PA4: 0   1   2   3   4   5   6   7
+//    PA3: X   Y   Z   Up  Dwn Lft Rgt Space
+//    PA2: P   Q   R   S   T   U   V   W
+//    PA1: H   I   J   K   L   M   N   O
+//    PA0: @   A   B   C   D   E   F   G
+//-------------------------------------------------
+
+static INPUT_PORTS_START( deluxecoco_keyboard )
```

Comment:
I can’t see this being so different to `coco_keyboard` that it warrants defining all the bits.  Use `PORT_INCLUDE(coco_keyboard)` and use `PORT_MODIFY` to change the rows that are actually different.

Reply:
Yesterday, when I made this change on Windows (GCC) I got a seg fault. But now I made the same change on my Mac and it is working fine. Later today I'll test this on Windows.